spec/helpers/for.js

```diff
@@ -10,6 +10,48 @@ describe('for helper', function() {
     // Build a test runner
     const runTestCases = testRunner({context});
 
+    it('should iterate once if zeroDiffStart is true.', function(done) {
```

Comment:
Would be more descriptive to say `includes the last element if includeLast is true`